.env.production.sample

```diff
@@ -0,0 +1,3 @@
+FOUNDRY_API_URL=<Fill in your Foundry instance base URL, e.g. https://example.palantir.com>
+APPLICATION_REDIRECT_URL=<Your deployed application domain>/auth/callback
```

Comment:
Should we word this similar to the other ones? "Fill in your ..." example https://example.app.com mentioned in the README